components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -336,52 +329,24 @@ open class GleanInternalAPI internal constructor () {
      */
     fun handleBackgroundEvent() {
         // Schedule the baseline and event pings
-        sendPingsInternal(listOf(BASELINE_STORE_NAME, "events"))
+        sendPings(listOf(Pings.baseline, Pings.events))
```

Comment:
We could do `Pings.baseline.send()` here, but that will log a debug message if the ping is empty when it's queued.  That's the only difference between the "public" API and this one.

Reply:
I think it's ok to print a message even though our pings are empty. I'm actually making this change in another PR: that would really, really help diagnosing if things go wrong with our pings as well.